packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -141,7 +141,7 @@ function removeIframe(iframe: HTMLIFrameElement) {
 
 interface BrowserEnvironmentOptions {
     distortionCallback?: DistortionCallback;
-    endowments?: object;
+    endowments: object;
```

Comment:
this is really the gist of the change. an alternative could be to move the endowments to be a required first argument, and moving options as the second argument, making it optional, that will work as well. nevertheless, this is a breaking change, so I'm good one way or the other.

Reply:
Why make it required what is expected to be provided? We had paths to just use an empty object if endowments weren't provided. Why not keep the fallback to the empty object?

Reply:
I'm fine with empty endowment, but that means you're not going to communicate between the two sides, which seems odd in principle. But probably we will have an easier path forward with that. I can make the changes.

Reply:
I have pushed the update with endowmnets being optional.